### PR TITLE
Features/compleate payment

### PR DESCRIPTION
**Pull request by adnansaadeddine**

### CODE REVIEW

File: components/Account/Order/AccountOrderConfirmShipping.vue [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+</script>
+
+<template>
+    <CheckoutConfirmCard
+        :title="$t('checkout.confirm.shipping.cardTitle')"
+        :subtitle="$t('checkout.confirm.shipping.cardSubtitle')"
+    >
+        <template v-if="method">
+            <CheckoutConfirmShippingMethod
+                :label="method.translated.name"
```
Comment (aheartforspinach) [inline components/Account/Order/AccountOrderConfirmShipping.vue:17]:
```suggestion
                :label="getTranslatedProperty(method, 'name')"
```

File: components/Account/Order/AccountOrderConfirmShipping.vue [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+                :deliveryTime="getShippingMethodDeliveryTime(method)"
+                :description="method.translated.description"
```
Comment (aheartforspinach) [inline components/Account/Order/AccountOrderConfirmShipping.vue:19]:
```suggestion
                :description="getTranslatedProperty(method, 'description')"
```

File: components/Account/Order/AccountOrderConfirmTerms.vue [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<script setup lang="ts">
+</script>
+
```
Comment (aheartforspinach) [inline components/Account/Order/AccountOrderConfirmTerms.vue:3]:
```suggestion
```

File: components/Checkout/Confirm/CheckoutConfirmShippingMethod.vue [hunk 4]
```diff
@@ -0,0 +1,31 @@
+<script setup lang="ts">
+defineProps<{
+    label: string;
+    deliveryTime?: string;
+    description?: string;
+    mediaUrl?: string;
+}>();
+</script>
+<template>
```
Comment (aheartforspinach) [inline components/Checkout/Confirm/CheckoutConfirmShippingMethod.vue:9]:
```suggestion
</script>

<template>
```

File: pages/account/order/edit/[id].vue [hunk 5]
```diff
@@ -0,0 +1,100 @@
+<script setup lang="ts">
+// useAuthentication().authenticate();
```
Comment (aheartforspinach) [inline pages/account/order/edit/[id].vue:2]: why is this commented out?

Comment (mloeffle) [reply pages/account/order/edit/[id].vue:2]: I think the authentication was/is worked on / @MorennMcFly updated it in another branch.
I think with the new branch this line would have to be:  `await useAuthentication().rerouteIfLoggedOut();`

File: pages/account/order/edit/[id].vue [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+};
+
+onMounted(async () => {
+    await loadOrderDetails();
+});
+</script>
+
+<template>
+    <div class="container">
+        <template v-if="order">
```
Comment (mloeffle) [inline pages/account/order/edit/[id].vue:45]: Maybe you can add a v-else containing a `SharedBanner` saying something along the lines 'your order could not be found. go back to your order overview an try again'. When the order search fails and the user misses the push notification they would be left with a white page

**Review by aheartforspinach — APPROVED** [review]

**Pull request merged**